### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MaxSchmide**

### CODE REVIEW

File: src/averageAges.js [hunk 1]
```diff
@@ -15,11 +15,18 @@
...
+    const inThisCentury = Math.ceil(person.died / 100) === century && isMale;
+
+    return century ? inThisCentury : isMale;
+  });
+
+  const ages = males.map(person => {
+    return person.died - person.born;
+  });
+
+  return +(ages.reduce((acc, cur) => acc + cur, 0) / ages.length).toFixed(2);
```
Comment (ilona-borkovska) [inline src/averageAges.js:29]: variable 'cur' does not tell us what it contains

File: src/averageAges.js [hunk 2]
```diff
@@ -55,7 +77,27 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const findMother = (name) => {
+    return people.find(person => person.name === name);
+  };
+
+  const poepleWithMother = people.filter(person => {
```
Comment (ilona-borkovska) [inline src/averageAges.js:84]: variable name has mistake

**Review by ilona-borkovska — APPROVED** [review]
Approve ✨

**Review by anstsot — CHANGES_REQUESTED** [review]
Good start:four_leaf_clover:
Let’s add a few fixes to improve your code.

**Review by anstsot — APPROVED** [review]
Good work, approving:sparkles:

**Pull request closed without merging**